pkg/operator/controller/ingress/deployment.go

```diff
@@ -180,6 +185,15 @@ func HardStopAfterIsEnabled(ic *operatorv1.IngressController, ingressConfig *con
 	return HardStopAfterIsEnabledByAnnotation(ingressConfig.Annotations)
 }
 
+// HAProxyTimespec returns the first substring from `input` that matches the
```

Comment:
Should we be more explicit here? Should it be exact vis-a-vis finding the first match?

Reply:
I plan to switch to using `clipHAProxyTimeoutValue()` for all the new fields, which should be more explicit in its parsing.